src/components/AboutUs/AddMember/AboutUsMember.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useEffect, useRef } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { ICreateTeamMember } from '@allTypes/reduxTypes/aboutUsStateTypes'
+import { Translation } from '@constants/translations'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { dispatch, useAppSelector } from '@redux/hooks'
+import { aboutUsMiddleware, aboutUsSelector } from '@redux/slices/aboutUs'
+import { Button } from '@uiComponents/Button'
+import FileField from '@uiComponents/FileField/FileField'
+import TextField from '@uiComponents/FormFields/TextField'
+import { createAboutUsValidationSchema } from '@validation/aboutUs/aboutUs'
+import Image from 'next/image'
+
+import { useImageUpload } from '../../../hooks/ImageUpload'
+
+export interface IAboutUsMemberProps {
+  setShowPersonForm: (value: boolean) => void
+}
+
+const defaultValues = {
+  header: '',
+  paragraph: '',
+  imageDescription: '',
+  imageURL: null,
+}
+const AboutUsMember = ({ setShowPersonForm }: IAboutUsMemberProps) => {
+  const { imageLoaded, handleFileChange } = useImageUpload()
+
+  const { isTeamMemberSubmitSuccess } = useAppSelector(aboutUsSelector.aboutUs)
+
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const [t] = useTranslation()
+
+  const handleClick = () => {
+    if (inputRef.current) {
+      inputRef.current.click()
+    }
+  }
+
+  const methods = useForm({
+    defaultValues,
+    resolver: yupResolver(createAboutUsValidationSchema),
+    mode: 'onChange',
+  })
+
+  const { handleSubmit, reset } = methods
+
+  const onSubmit = (data: ICreateTeamMember) => {
+    dispatch(aboutUsMiddleware.createTeamMember(data))
+  }
+
+  useEffect(() => {
+    if (isTeamMemberSubmitSuccess) {
+      dispatch(aboutUsMiddleware.resetCreateTeamMemberSubmitSuccess())
+      setShowPersonForm(false)
+      reset(defaultValues)
+    }
+    //   eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isTeamMemberSubmitSuccess])
+
+  return (
+    <div>
+      <FormProvider {...methods}>
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className="w-full"
+        >
+          <div className="mb-10 flex w-full flex-col justify-between xl:flex-row-reverse">
+            <div className="space-y-4">
+              <div>
+                {imageLoaded ? (
+                  <Image
+                    src={imageLoaded}
+                    loader={() => imageLoaded}
+                    width={500}
+                    height={680}
+                    alt="picture"
+                    className="h-[345px] w-[500px]"
+                  />
+                ) : null}
+              </div>
+              <div
+                className={`${
+                  imageLoaded !== undefined ? 'flex flex-col-reverse' : 'flex w-full justify-center'
+                }`}
```

Comment:
imageLoaded
```suggestion
                  !imageLoaded ? 'flex flex-col-reverse' : 'flex w-full justify-center'
                }`}
```

Reply:
We should never have undefined in our defined variables

Reply:
done